src/test/java/list_exercises_tests/CompareTest.java

```diff
@@ -0,0 +1,124 @@
+package list_exercises_tests;
+import list_excercises.service.Compare;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.internal.matchers.Null;
+
+import javax.print.DocFlavor;
```

Comment:
If you accidentally add some wrong import, remember to delete it after. It's good to keep the code clean. There's an option in IntelliJ which is organise imports (there's a shortcut cmd+option+O - dont know if it's gonna work for you though).